components/ldap.rst

```diff
@@ -0,0 +1,66 @@
+.. index::
+   single: Ldap
+   single: Components; Ldap
+
+The Ldap Component
+==================
+
+    The Ldap component provides a means to connect to an LDAP server (OpenLDAP or Active Directory).
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/ldap`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/ldap).
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+The :class:`Symfony\\Component\\Ldap\\LdapClient` class provides methods
+to authenticate and query against an LDAP server.
+
+The :class:`Symfony\\Component\\Ldap\\LdapClient` class can be configured
+using the following options:
+
+* ``host``: IP or hostname of the LDAP server
+
+* ``port``: Port used to access the LDAP server
+
+* ``version``: The version of the LDAP protocol to use
+
+* ``useSsl``: Whether or not to secure the connection using SSL
+
+* ``useStartTls``: Whether or not to secure the connection using StartTLS
+
+* ``optReferrals``: Specifies whether to automatically follow referrals
+  returned by the LDAP server
```

Comment:
Sorry, I didn't notice during my last review, but I think having a definition list would better fit what we are trying here:

```rst
``host``
    IP or hostname of the LDAP server

``port``
    Port used to access the LDAP server

``version``
    The version of the LDAP protocol to use

``useSsl``
    Whether or not to secure the connection using SSL

``useStartTls``
    Whether or not to secure the connection using StartTLS

``optReferrals``
    Specifies whether to automatically follow referrals returned by the LDAP server
```